trickle test pass condition is backwards?

See https://github.com/web-platform-tests/wpt/pull/52429/checks?check_run_id=41905153312:

```
=================================== FAILURES ===================================
___________________________ TestTrickle.test_trickle ___________________________

self = <functional.test_pipes.TestTrickle testMethod=test_trickle>

    def test_trickle(self):
        #Actually testing that the response trickles in is not that easy
        t0 = time.time()
        resp = self.request("/document.txt", query="pipe=trickle(1:d2:5:d1:r2)")
        t1 = time.time()
        with open(os.path.join(doc_root, "document.txt"), 'rb') as f:
            expected = f.read()
        self.assertEqual(resp.read(), expected)
>       self.assertGreater(6, t1-t0)
E       AssertionError: 6 not greater than 6.26174259185791

wptserve/tests/functional/test_pipes.py:156: AssertionError
------------------------------ Captured log call -------------------------------
INFO     root:server.py:867 Starting http server on http://localhost:37863
INFO     root:server.py:885 Stopped http server on localhost:37863
```

`/document.txt` is 24 bytes, so the [trickle pipe](https://web-platform-tests.org/writing-tests/server-pipes.html#trickle) should be:

1. Send one byte (1/24)
2. Wait two seconds (total: 2s)
3. Send five bytes (6/24)
4. Wait one second (total: 3s)
5. Send five bytes (11/24)
6. Wait one second (total: 4s)
5. Send five bytes (16/24)
6. Wait one second (total: 5s)
5. Send five bytes (21/24)
6. Wait one second (total: 6s)
5. Send five bytes (26/24, so EOF)

This implies that the total time should be expected to be greater than six seconds, if I'm not mistaken?

Running this locally, I'm getting `t1 - t0` equal to ~0.01, which definitely isn't indicative of any trickling happening!

The problem I think is that we're not actually measuring the `resp.read()` call.

--- a/tools/wptserve/tests/functional/test_pipes.py
+++ b/tools/wptserve/tests/functional/test_pipes.py
@@ -146,14 +146,16 @@ server: http://localhost:{0}""".format(self.server.port).encode("ascii")
 
 class TestTrickle(TestUsingServer):
     def test_trickle(self):
-        #Actually testing that the response trickles in is not that easy
-        t0 = time.time()
-        resp = self.request("/document.txt", query="pipe=trickle(1:d2:5:d1:r2)")
-        t1 = time.time()
-        with open(os.path.join(doc_root, "document.txt"), 'rb') as f:
+        # Actually testing that the response trickles in is not that easy
+        clock_info = time.get_clock_info("monotonic")
+        t0 = time.monotonic()
+        with self.request("/document.txt", query="pipe=trickle(1:d2:5:d1:r2)") as resp:
+            actual = resp.read()
+        t1 = time.monotonic()
+        with open(os.path.join(doc_root, "document.txt"), "rb") as f:
             expected = f.read()
-        self.assertEqual(resp.read(), expected)
-        self.assertGreater(6, t1-t0)
+        self.assertEqual(actual, expected)
+        self.assertGreater(t1 - t0, 6 - clock_info.resolution)
 
     def test_headers(self):
         resp = self.request("/document.txt", query="pipe=trickle(d0.01)")

